docs/core/idioms/targets.md

```diff
@@ -0,0 +1,57 @@
+# Using a Result target
```

Comment:
Looking for a way to make this more of a specific idiom if necessary

Reply:
Maybe you can name this "Using Result targets for efficient caching" or something like that?